lms/djangoapps/notes/tests.py

```diff
@@ -39,11 +42,53 @@ def test_notes_enabled(self):
         Tests that notes are enabled when the course tab configuration contains
         a tab with type "notes."
         '''
-        self.course.tabs = [{'type': 'foo'},
-                            {'name': 'My Notes', 'type': 'notes'},
-                            {'type': 'bar'}]
+        with self.settings(FEATURES={'ENABLE_STUDENT_NOTES': True}):
```

Comment:
The `django.test.TestCase` offers it up as a way of overriding settings inside a test: https://docs.djangoproject.com/en/1.4/topics/testing/#overriding-settings

It's a little safer to use `patch.dict` on feature flags, but this is such a small test that using `patch.dict` seemed like overkill.